windborne/data_api.py

```diff
@@ -549,9 +549,34 @@ def get_flying_missions(output_file=None, print_results=False):
         dict: The API response containing list of flying missions.
     """
 
+    # Initial query to get total flying
+    query_params = {
+        'short': 'true',
+        'range': 'flying',
+        'include_total': 'true',
+        'page': 0,
+        'page_size': 32
+    }
+
     url = f"{DATA_API_BASE_URL}/missions.json"
-    flying_missions_response = make_api_request(url)
+    flying_missions_response = make_api_request(url, params=query_params)
+    total_flying = flying_missions_response.get('total', 0)
+    del query_params['include_total']
```

Comment:
Because we didn't get the total back, we should just keep fetching until mission count < page size. Sure, we'll have a 1/64 chance of making an extra request, but that's fine